### PR TITLE
URL's updated in the sample-apps and three images added to the docs f…

### PR DESCRIPTION
**Pull request by finalelement**
URL's updated for the download and three images added to the Docs.

Please do check out what is the reason of existence behind this line "/clara_pt_deepgrow_2d_annotation/versions/1/files/models/model.pt" in the main file. I am not sure.

Signed-off-by: vnath <vnath@nvidia.com>

### CODE REVIEW

**Review by SachidanandAlle — CHANGES_REQUESTED** [review]
Path is not correct.. have to remove 2nd line

File: sample-apps/deepgrow_left_atrium/main.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def __init__(self, app_dir, studies):
                     os.path.join(self.model_dir, "deepgrow_2d.pt"),
                     os.path.join(self.model_dir, "deepgrow_2d_final.pt"),
                 ],
-                "url": "https://api.ngc.nvidia.com/v2/models/nvidia/med"
+                "url": "https://www.dropbox.com/s/bx832eozn8rcuyg/deepgrow_2d_left_atrium.pt?dl=1"
                 "/clara_pt_deepgrow_2d_annotation/versions/1/files/models/model.pt",
```
Comment (SachidanandAlle) [inline sample-apps/deepgrow_left_atrium/main.py:28]: Remove this line

Comment (finalelement) [reply sample-apps/deepgrow_left_atrium/main.py:28]: Noted

**Comment by finalelement** [comment]
Changes Done

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**